### ISSUE TITLE
touch.y coordinate is wrong on windows  [$100] 

### ISSUE THREAD

**Issue by grmcom** [kivy/kivy#4309]
There is an issue with touch.y coordinate on windows which makes software appear unreliable when attempting to touch small widgets.

In fullscreen the touch.y is about 30 pixels higher  than the actual touch point over the whole screen.

Fullscreen - Touch at top of screen:
![fullscreen top of screen](https://cloud.githubusercontent.com/assets/329604/15535823/e712b3aa-2266-11e6-9788-c53876c181f1.jpg)

Fullscreen - Touch at middle of screen:
![fullscreen middle of screen](https://cloud.githubusercontent.com/assets/329604/15535822/e71158f2-2266-11e6-84d1-0acb973d8aa9.jpg)

Fullscreen - Touch at bottom of screen
![fullscreen bottom of screen](https://cloud.githubusercontent.com/assets/329604/15535824/e712b9a4-2266-11e6-9db9-d7dc1abe53d6.jpg)

In windowed mode the touch.y is slightly below the actual touch point at the top of the screen.
touch.y is correct around 1/4 of the way down the screen.
touch.y is around 35 pixels above the actual touch position at the bottom of the screen.

Windowed - Touch at top of screen
![windowed top of screen](https://cloud.githubusercontent.com/assets/329604/15535825/e714d7e8-2266-11e6-8610-365c70b071ea.jpg)

Windowed - Touch at middle of screen
![windowed middle of screen](https://cloud.githubusercontent.com/assets/329604/15535827/e7157a90-2266-11e6-9892-ec42e554ccc2.jpg)

Windowed - Touch at bottom of screen
![windowed bottom of screen](https://cloud.githubusercontent.com/assets/329604/15535826/e7151fb4-2266-11e6-82be-077e8960661c.jpg)

This issue can make software appear unreliable as users think they are touching a widget, but the widget does not respond as the y.coordinate is offset.
This is very noticeable on large screens.

**Comment by grmcom** [comment]
This issue has been observed on Windows 8.1 and Windows 10 using an  Acer T272HL and an Asus PT201Q.

**Comment by grmcom** [comment]
$100 bounty on bountysource
https://www.bountysource.com/issues/34557886-touch-y-coordinate-is-wrong-on-windows

**Comment by dessant** [comment]
Is the bug also present in Kivy master?

**Comment by grmcom** [comment]
I have just installed the nightly wheel.
The issue in fullscreen mode has been resolved.
However the problem still remains in windowed mode.

**Comment by werton** [comment]
I have absolutely the same issue on my Asus Transformer Book T100TAM (windows 8.1) on 1.9.0, 1.9.1 and last dev version of kivy.

**Comment by matham** [comment]
I believe I fixed it. I made two PRS solving it in two ways. https://github.com/kivy/kivy/pull/4317 and https://github.com/kivy/kivy/pull/4318. The second one is closer to how it's done in SDL2, but I'm not convinced SDL2 does it correctly in all conditions. They should be identical except for when the window is resizable but is also borderless.

I don't have a very large touch screen. To figure out which one is correct, it needs to be tested on a large screen where potential issues are more apparent.
